### PR TITLE
Use existing sliderblock scale

### PR DESCRIPTION
**Pull request by bierik**
Closes https://github.com/4teamwork/ftw.sliderblock/issues/20

### CODE REVIEW

File: ftw/sliderblock/browser/templates/sliderblock.pt [hunk 1]
```diff
@@ -23,7 +23,7 @@
                             <div class="sliderImage"
                                  tal:define="images pane/@@images">
                                  <img tal:define="scales pane/@@images;
-                                                  thumbnail python: scales.scale('image');"
+                                                  thumbnail python: scales.scale('image', scale='sliderblock', direction='down');"
```
Comment (jone) [inline ftw/sliderblock/browser/templates/sliderblock.pt:26]: Where is the scale `sliderblock` configured? Cannot find it.

Comment (jone) [reply ftw/sliderblock/browser/templates/sliderblock.pt:26]: @bierik ping 😏

Comment (bierik) [reply ftw/sliderblock/browser/templates/sliderblock.pt:26]: https://github.com/4teamwork/ftw.slider/blob/master/ftw/slider/profiles/default/propertiestool.xml#L16

Comment (jone) [reply ftw/sliderblock/browser/templates/sliderblock.pt:26]: that's fine 👍 thanks

**Pull request merged**